src/leiningen/i18n/bin/remove-line-numbers.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+# This script's only function is to strip the line numbers from the location
+# comments in a PO or POT file. It does this using a sed one-liner at the
+# bottom. The preceding lines are largely about checking for error conditions.
+
+set -eo pipefail
+
+gettext_file="$1"
+
+set -u # there should be no more undefined variables
+
+if [[ -z "$gettext_file" ]]; then
+  echo "ERROR: no PO or POT file given" 1>&2
+  echo "usage: $0 <PO-or-POT-file>" 1>&2
+  exit 1
+fi
+
+if [[ ! -e "$gettext_file" ]]; then
+  echo "ERROR: the file '$gettext_file' does not exist" 1>&2
+  exit 1
+fi
+
+if [[ ! -r "$gettext_file" ]]; then
+  echo "ERROR: don't have permission to open '$gettext_file'" 1>&2
+  exit 1
+fi
+
+sed -i -e "/^#:.*[0-9]\+\s*$/ s/:[0-9]\+//g" "$gettext_file"
```

Comment:
So I realized why this had duplicate entries in the commit: https://github.com/puppetlabs/classifier/commit/cf48c8d5489cd3502046472c9d268da8517b235b#diff-99771c4194f9112cce9eb1761c211a91R77

Mainly because this just removes the line #s as stated. However when there are line numbers present there are multiple file path entries.  When used on gettext that only puts the file location - it's a de-duped path.  I don't really care, but I just want to make sure we feel like this will meet the objective of keeping the POT churn at a minimum. I think if they work in a consistent manner across all Jenkins' machines - it should just be a bit of chaff in the first commit, and only add lines when there's an additional entry.  And then have consistent dup entries from here on out? Non issue? With additional (when there are multiple) string entries (in a single file) you'll have additional noise in the POT that you otherwise wouldn't have.